README.md

```diff
@@ -335,8 +267,8 @@ following (non-exhaustive) set of useful options:
 
 - ``--dry-run``: Perform a dry run build. This is off by default.
 - ``--test``: Test the toolchain after it has been compiled. This is off by default.
-- ``--gpu`` (Linux only): Build with GPU support. This is off by default.
 - ``--pkg`` (macOS only): Build a toolchain installer package (`.pkg`). This is off by default.
+- ``-x10``: Built with x10 support.
```

Comment:
--x10?

Reply:
Oops, I should just remove this, because it's deprecated now. Removed.